src/sys/uio.rs

```diff
@@ -56,6 +59,38 @@ pub fn pread(fd: RawFd, buf: &mut [u8], offset: off_t) -> Result<usize>{
     Errno::result(res).map(|r| r as usize)
 }
 
+// process_vm_{read,write}v currently doesn't use flags, but we define this
```

Comment:
See the earlier discussion on the PR - I'm pretty indifferent here, @kamalmarhubi expressed a preference to just add the flags argument now. I agree that probably there will never be flags.

I guess the other option is that we just break names from libc slightly (which we do in other places) and add a four-parameter `process_vm_readv_flags(pid, local_iov, remote_iov, flags)` if we ever need to, which avoids the API break.